src/wiki/help-pages/Change-Themes.md

```diff
@@ -0,0 +1,135 @@
+
+### Themes Currently Available
+
+[![Breeze Dark](https://img.shields.io/badge/Breeze--Dark-blue?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Breeze-Dark.zip)
+[![Catppuccin Frappe](https://img.shields.io/badge/Catppuccin--Frappe-EF9F76?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Frappe.zip)
+[![Catppuccin Latte](https://img.shields.io/badge/Catppuccin--Latte-DC8A78?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Latte.zip)
+[![Catppuccin Macchiato](https://img.shields.io/badge/Catppuccin--Macchiato-A6DA95?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Macchiato.zip)
+[![Catppuccin Mocha](https://img.shields.io/badge/Catppuccin--Macchiato-DDB6F2?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Mocha.zip)
+[![Dracula](https://img.shields.io/badge/Dracula-BD93F9?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Dracula.zip)
+[![Fluent Dark](https://img.shields.io/badge/Fluent--Dark-60CDFF?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Fluent-Dark.zip)
+[![Nord](https://img.shields.io/badge/Nord-88C0D0?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Nord.zip)
+
+### Themes Available only on Discord for now
+
+[![Borest Theme](https://img.shields.io/badge/Borest--Theme-1E1E2E?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1033569391931047957)
+[![Rangoon Lighting](https://img.shields.io/badge/Rangoon--Lighting-FFBF22?style=for-the-badge&logo=discord&logoColor=333333)](https://discord.com/channels/1031648380885147709/1033899002644668456)
+[![Dark Legacy Theme](https://img.shields.io/badge/Legacy--Dark-32373C?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1032990022447661096)
+[![Steam Classic](https://img.shields.io/badge/Steam--Classic-4C5844?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1033097334927654952)
+[![Deep Dark Theme](https://img.shields.io/badge/Deep--Dark--Theme-141414?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1033025016910053436)
+
+### Icons packs only on Discord for now
+
+[![Fluent Icons](https://img.shields.io/badge/Fluent--Icons-yellow?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1033100137121194014)
```

Comment:
We should add this themes to prismlauncher/themes instead

Reply:
Do we just add a icons folder in the repo?

Reply:
We will discuss this on discord is that ok?

Reply:
We can do this in another PR. No probs! It's good for now.